pina/geometry/location.py

```diff
@@ -14,4 +14,12 @@ def sample(self):
         Abstract method for sampling a point from the location. To be
         implemented in the child class.
         """
+        pass
+
+    @abstractmethod
+    def is_inside(self, point, check_border=False):
+        """
```

Comment:
update doc

Reply:
I'm sorry but I don't follow. What do you mean? I have the docstrings:  
        """
        Abstract method for checking if a point is inside the location. To be
        implemented in the child class.
        """

Reply:
you need to specify what point, check_border are

Reply:
Oh gotcha. Just added.